### PR TITLE
Add Rusty Days talk

### PR DESCRIPTION
**Pull request by LukeMathWalker**
I spent quite a large portion of my talk at Rusty days discussing logging/structured logging/distributed tracing, covering our usage of `tracing`. 
If you think it's relevant I'd be happy to add it to the list :grin: 

P.S. there were some trailing whitespaces that my VIM setup ruthlessly eliminated. Let me know if you want me to leave them there.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -424,10 +424,13 @@ Tracing.
 
 * [Bay Area Rust Meetup talk and Q&A][bay-rust-2018-03], March 2018
 * [RustConf 2019 talk][rust-conf-2019-08-video] and [slides][rust-conf-2019-08-slides], August 2019
+* Are we observable yet? @ RustyDays - [talk][rusty-days-2020-08-video] and [slides][rusty-days-2020-08-slides], August 2020
```
Comment (hawkw) [inline README.md:427]: nit: for the other two entries here, i made the title of the talk link to the video. maybe we should do that here, as well, for consistency?

**Comment by hawkw** [comment]
as a side note, i wonder if we should add the list of resources to the `tracing` crate’s README and/or lib.rs docs, so it’s easier to find? cc @davidbarsky

**Comment by hawkw** [comment]
surprised ci is failing for this — maybe new warnings?

**Comment by hawkw** [comment]
Okay, we just merged PR #882 that should fix the CI failure. Mind rebasing onto that?

File: README.md (hunk 1 above)
Comment (LukeMathWalker) [reply README.md]: Done :heavy_check_mark:

**Comment by LukeMathWalker** [comment]
Rebased :+1:

**Comment by LukeMathWalker** [comment]
Thank you :grin:

**Pull request merged**